.gitignore

```diff
@@ -0,0 +1 @@
+/node_modules/
```

Comment:
Nit - Changing the path to `**/node_modules` might work here. Would it be possible to make a change to gitignore the node_modules and remove them from the git history before merge? They can get pretty large.